src/patterns/atoms/MovieDate/MovieDate.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import "./MovieDate.scss"
```

Comment:
lets try to add CSS Modules to make it more real-world application creation https://webpack.js.org/loaders/css-loader/#modules

We can discuss it during our session

Reply:
(not required, but nice to have and practice)

Reply:
:( not implemented jet I will